budget/__main__.py

```diff
@@ -173,6 +199,12 @@ def readArgs():
         if len(sys.argv) != 3:
             raise UserWarning("Missing transaction parameters for '{0}' command.".format(sys.argv[1]))
         return sys.argv[1], sys.argv[2]
+    elif sys.argv[1] == "edit":
+        if sys.argv[2] not in ('income', 'expense'):
+            raise UserWarning("Invalid command. Edit command has to have 'expense' or 'income' as 3rd argument.")
+        if len(sys.argv) != 5:
```

Comment:
This could as well mean more than 5 arguments have been provided. Let's not say something is missing, but instead there must be exactly 5 arguments.